templates/ros2_control/controller/dummy_chainable_controller.cpp

```diff
@@ -238,16 +268,25 @@ controller_interface::return_type DummyClassName::update_reference_from_subscrib
 controller_interface::return_type DummyClassName::update_and_write_commands(
   const rclcpp::Time & time, const rclcpp::Duration & /*period*/)
 {
+  auto current_ref = input_ref_.readFromRT();
+  const auto age_of_last_command = get_node()->now() - (*current_ref)->header.stamp;
```

Comment:
why is this here? This should be in the update reference from subscribers only. Please adjust